packages/default/scss/menu/_layout.scss

```diff
@@ -116,6 +116,14 @@ $image-spacing: 8px !default;
         .k-item {
             display: block;
             border-color: inherit;
+
+            .k-icon {
+                margin-right: $image-spacing;
```

Comment:
If the `margin-right` style is inside "menu" file. Why the relevant RTL styles are in the "grid" file?

Reply:
The margin-left and right in the 'grid' file is resetting the margins in RTL mode for the column menu within the grid, which are set as:

    .k-columnmenu-item {
        padding: $nav-item-padding-y $nav-item-padding-x;
        cursor: pointer;

        > .k-icon {
            margin-right: $icon-spacing;
        }
    }
  
While the styles set in the 'menu' file are targeting the menu outside of the grid.